.github/workflows/release.yml

```diff
@@ -20,7 +20,11 @@ jobs:
           cache: npm
           registry-url: "https://registry.npmjs.org"
       - run: npm ci --workspaces
-      - run: npm run build --if-present --workspaces
+      # Running top-level ci should happen after workspaces level, running these
+      # the other way around results in top-level dependencies (namely, lerna)
+      # not being installed.
+      - run: npm ci
+      - run: npm run build
```

Comment:
Release runs build as part of it, or, at least should. Part of publishing a package is running the build step iirc